packages/core/src/models/promptVariants.ts

```diff
@@ -24,25 +24,26 @@ export class PromptVariants {
     this.variant = foundVariant;
   }
 
-  async setCurrent() {
-    const template = this.format();
-    const hash = Md5.hashStr(template);
+  async setCurrent(variant: string, raw = false) {
+    const template = this.format(raw);
+    const { slug } = this.definition;
+    const hash = `${slug}-${variant}-${Md5.hashStr(template)}`;
+    console.log("*** setCurrent ***");
+    console.log("Hash", hash);
+    console.log("Slug", slug);
+    console.log("Variant", variant);
+    console.log("-----");
     const existing = await this.prisma.prompt.findFirst({
       where: {
-        hash,
+        AND: [{ hash }, { slug }, { variant }],
```

Comment:
Perhams an index on these?

Reply:
Though ... if we're caching now ...

Reply:
It's quite a small table - not sure it's worth the hassle of doing a migration! And yes - I'll be adding the prompt table to the Accelerate cache